src/util/message.h

```diff
@@ -44,8 +46,51 @@ struct messaget
     template <typename... Args>
     static void println(FILE *f, VerbosityLevel lvl, Args &&...args)
     {
-      if(lvl == VerbosityLevel::Error)
-        fmt::print(f, "ERROR: ");
+      /** 
+		* Newly added code to address issue #886.
+		* It prepends a well formatted timestamp and log level to each LOG message. 
+		* It is transparent to the existing logging function, i.e., log_message.
+		* TODO: It can be implemented using a separate function and invoked here and/or elsewhere when necessary. 		
+		*/
```

Comment:
Fix the indentation of this comment block. I suggest replacing it with `//` so clang-format doesn't think this is a doxygen comment.